src/main/java/com/coveros/selenified/utilities/Sauce.java

```diff
@@ -53,6 +59,44 @@ public static Boolean isSauce() {
         return hub != null && hub.contains("ondemand.saucelabs.com");
     }
 
+    /**
+     * TODO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1135)